### DISCUSSION TITLE
Folding isn’t working. How to fold?

### DISCUSSION THREAD

**Discussion by kerim** [vslinko/obsidian-outliner#71] [Q&A] [answered]
Maybe it just doesn't work with the latest insider build, but I have installed this and I'm not able to use it. I was wondering if there are some instructions to get started, just in case I'm doing something wrong?

**Comment by danieltomasz** [comment, marked as answer]
Did you activate the plugin in the Obsidian plugin pane?
The latest build is now released as Obsidian 1.1.
Here is a really nice  outline of the steps
https://github.com/vslinko/obsidian-outliner#how-to-install

Do your problem persist after you followed these steps?

**Reply by kerim** [reply]
The only issue I could find was that I was using an older version of the plugin (despite having only installed it yesterday). I forced it to update and it is now version 1.1.1 (did you just push an update?) It has applied the styling, but I can't fold or use the command keys as per the instructions. (I'm on MacOS using an M1 Macbook, in case that is relevant...)

**Reply by vslinko** [reply]
@kerim Hi! Please check that folding is enabled in Obsidian settings.
<img width="1520" alt="Screenshot 2021-04-13 at 22 01 31" src="https://user-images.githubusercontent.com/205823/114606354-d6aa0100-9ca3-11eb-930b-582cc6e94557.png">

**Reply by vslinko** [reply]
@kerim Added notification for this case in 1.1.2 https://github.com/vslinko/obsidian-outliner/releases/tag/1.1.2

**Reply by kerim** [reply]
Ahhh. I see. I thought that was a feature of the plugin, not part of Obsidian. Working now. Thanks.
